docs/datastores.md

```diff
@@ -0,0 +1,59 @@
+Datastores
+==========
+
+Version 1.0+ supports using Postgres as a storage backend instead of the
+legacy NeDB storage backend. If you wish to take advantage of this, please
+read on.
+
+NeDB EOL
+--------
+
+NeDB is a library which is used to store json documents locally to disk to give the bridge some local persistent state.
+Until 1.0, it was the only way to store this information. 1.0 brings in support for PostgreSQL.
+
+After version `1.0`, NeDB will also be deprecated and shouldn't be used for new installations. NeDB is
+unmaintained[1] and doesn't scale well for the needs of this bridge. Changes such as **puppeting** will
```

Comment:
I'm not sure why puppeting is bold here - it doesn't appear to be adding to the discussion? Also let's call them features.
```suggestion
unmaintained[1] and doesn't scale well for the needs of this bridge. Future features such as puppeting will
```